src/cli/repl/core.ts

```diff
@@ -11,13 +11,40 @@ import { commandNames, getCommand, standardReplOutput } from './commands'
 import * as readline from 'node:readline'
 import { splitAtEscapeSensitive } from '../../util/args'
 import { executeRShellCommand } from './commands/execute'
+import { guard } from '../../util/assert'
+import { scripts } from '../common/scripts-info'
+import type { OptionDefinition } from 'command-line-usage'
 
 const replCompleterKeywords = Array.from(commandNames, s => `:${s}`)
 
 /**
  * Used by the repl to provide automatic completions for a given (partial) input line
  */
 export function replCompleter(line: string): [string[], string] {
+	const singleCommand = replCompleterKeywords.find(k => (k + ' ') === line)
+	if(singleCommand !== undefined){
+		const applicableCommand = getCommand(singleCommand.substring(1))
```

Comment:
Wieso der substring?

Reply:
Weil die gespeicherten Commands noch nicht mit dem ":" beginnen sondern direkt anfangen

Reply:
Wäre hier ein "singleCommand.replace(':', '')" besser für das Verständnis?

Reply:
ne ich würds auslagern in ein `toCommand` oder so :D uuuund vlt noch nen guard ob es wirklich mit `:` beginnt!